cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/UserPaymentMethodDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.cloudesire.platform.apiclient.dto.model.dto;
+
+public class UserPaymentMethodDTO extends BaseEntityDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2160)